test/spmd/test_xla_sharding.py

```diff
@@ -48,11 +48,11 @@ def test_xla_shards(self):
       self.assertEqual(shard.data.shape, (shard_len,))
       start, end = (i, i + 1) * shard_len
       expected = torch.arange(start, end, dtype=torch.float32)
-      self.assertTrue(torch.allclose(shard.data.cpu(), expected))
+      self.assertTrue(torch.allclose(shard.data, expected))
```

Comment:
can we add a assert above to check that `shard.data` is on cpu device. Just to make check more explict.

Reply:
Good point, added